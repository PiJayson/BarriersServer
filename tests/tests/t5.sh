#!/bin/sh

. common

$SCRIPTS/errno2.sh 

