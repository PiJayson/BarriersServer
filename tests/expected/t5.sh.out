Init barrier id: 7 width: 2 ... result 0
Calling wait, barrier id: 7
Waiting done, barrier id: 7 ... result: -1 errno: 302
